recipes/ixwebsocket/all/conanfile.py

```diff
@@ -56,13 +57,16 @@ def config_options(self):
             del self.options.use_openssl
 
     def requirements(self):
+        if (self._can_use_openssl() and not self.options.use_mbed_tls and self.options.use_tls
+                and (self.options.get_safe("use_openssl") or self.settings.os == "Linux" 
+                     or self.settings.os == "Windows" and not self.options.get_safe("openssl"))): # OpenSSL jiggling with Windows necessary on newer versions.
```

Comment:
There is no `openssl` option.

Also, can we add a parenthesis here, not sure what is the intention of the last `and`...